lib/compat/wordpress-5.9/theme.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Additions to the WordPress theme.php file
+ *
+ * @package gutenberg
+ */
+
+if ( ! function_exists( 'wp_is_block_theme' ) ) {
+	/**
+	 * Returns whether the current theme is an FSE theme or not.
+	 *
+	 * @return boolean Whether the current theme is an FSE theme or not.
```

Comment:
Wording: _block_ theme instead of _FSE_ theme

```suggestion
	 * Returns whether the current theme is a block theme or not.
	 *
	 * @return boolean Whether the current theme is a block theme or not.
```